src/app/aria-selected.directive.ts

```diff
@@ -0,0 +1,27 @@
+// AriaSelectedDirective - An attribute directive that uses the 'ariaselected' selector to 
+// to apply the aria-selected attribute and value when the item gains and loses focus. 
+import { Directive, ElementRef, Renderer2, HostListener } from '@angular/core';
+@Directive({
+    selector: '[ariaselected]'
```

Comment:
I think this should be prefixed with your appname.  The name is too close to the existing html css attribute that someone reading the html will be confused, and runs the risk of conflicting with future css spec changes (like if they decide to remove the hyphen).